source/gameengine/Converter/KX_ConvertActuators.cpp

```diff
@@ -977,7 +977,7 @@ void BL_ConvertActuators(const char* maggiename,
 			}
 
 			tmp = new SCA_2DFilterActuator(gameobj, filtermode,  _2dfilter->flag,
-			                               _2dfilter->float_arg, _2dfilter->int_arg,
+			                               _2dfilter->float_arg, _2dfilter->int_arg, _2dfilter->mipmap,
```

Comment:
It Is an ont no in filter constructor? I can't check. I'm at Montpellier. I'll see tomorrow